StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/ReadabilityRules/SA1134CodeFixProvider.cs

```diff
@@ -29,7 +30,7 @@ internal class SA1134CodeFixProvider : CodeFixProvider
         /// <inheritdoc/>
         public override FixAllProvider GetFixAllProvider()
         {
-            return CustomFixAllProviders.BatchFixer;
```

Comment:
❓ Any idea what's making this not deterministic? We originally added the custom batch fixer to address non-deterministic behavior in the one provided by Roslyn.

Reply:
No. I had the test fail exactly once in the IDE. I have not been able to reproduce it since.